### DISCUSSION TITLE
How do I fix this?

### DISCUSSION THREAD

**Discussion by SuPeRDuDe5898** [ppy/osu#26429] [Q&A] [answered]
I want to make a Sonic the Hedgehog ranked map but it looks like I need to fix these. Is there a way to do so?
![image](https://github.com/ppy/osu/assets/155716165/44b865ae-c255-44b8-91a9-fa8a4fe58813)

**Comment by SuPeRDuDe5898** [comment]
I can figure out how to change the resolution, but how do I add hitsounds, and have 30 seconds drain time?

**Comment by nagi-desuuu** [comment]
Have you tried going to the mapping communities like reddit and discord and finding the solution yourself?

**Reply by SuPeRDuDe5898** [reply]
I probably won't figure it out that easily, so that's why I kinda want a direct answer.

**Comment by bdach** [comment, marked as answer]
> There are no hitsounds

Shown because none of the objects that can be hitsounded (circles / sliders) use the added sounds (whistle / finish / clap). Refer to [ranking criteria](https://osu.ppy.sh/wiki/pl/Ranking_criteria#rules.4).

> Less than 30 seconds of drain time

Basically means the map is too short. See [ranking criteria](https://osu.ppy.sh/wiki/pl/Ranking_criteria#rules.1).

**Reply by SuPeRDuDe5898** [reply]
oh, i can easily fix it then, cuz im making a beatmap using the FULL version of "I'm Here" from sonic frontiers, so it'll be way longer than 30 seconds, and the hitsounds i can do with a single click
thanks!
